### ISSUE TITLE
Is this properly working with latest SvelteKit?

### ISSUE THREAD

**Issue by MelodicCrypter** [vite-pwa/sveltekit#33]
My versions:

```
"@sveltejs/kit": "^1.11.0",
"@vite-pwa/sveltekit": "^0.1.3",
"vite-plugin-pwa": "^0.14.4",
```

I'm not sure if this is still working. It says it cannot find manifest. I mean, based on the [doc](https://vite-pwa-org.netlify.app/frameworks/sveltekit.html), I followed each instruction but when I ran it it seems the PWA is not working. Anyone can help? Thank you.

**Comment by userquin** [comment]
@MelodicCrypter no, check WIP PR here: https://github.com/vite-pwa/sveltekit/pull/32

**Comment by MelodicCrypter** [comment]
@userquin oh I see. Good to know that there's a PR on going. Thanks!

**Comment by MelodicCrypter** [comment]
@userquin is there a way I can use your PR in my local sveltekit project? I mean, I don't know how to properly implement it. Should I just clone your pr then build it locally? Then import it in my local project? Can you give some proper direction while waiting for the merge? Thanks.

**Comment by userquin** [comment]
@MelodicCrypter I'm very busy, you can fork PR branch and build it in your local (`pnpm run build && pnpm pack`) and use the tar.gz, I Will try to release it asap.

**Comment by userquin** [comment]
@MelodicCrypter the docs can be found here (also WIP PR, it will be merged once published v0.2.0) https://github.com/vite-pwa/docs/pull/32, check docs preview for kit here https://deploy-preview-32--vite-pwa-org.netlify.app/frameworks/sveltekit.html

**Comment by MelodicCrypter** [comment]
Thank you @userquin!
